src/atomate2/vasp/builders/magnetism.py

```diff
@@ -0,0 +1,23 @@
+"""Module defining VASP magnetic orderings builder."""
+
+from __future__ import annotations
+
+from atomate2.common.builders.magnetism import (
+    MagneticOrderingsBuilder as MagneticOrderingsBuilderBase,
+)
+from atomate2.vasp.schemas.magnetism import (
+    MagneticOrderingOutput,
+    MagneticOrderingRelaxation,
+)
+
+
+class MagneticOrderingsBuilder(MagneticOrderingsBuilderBase):
```

Comment:
It's really nice to see how few additions were required to make this work with VASP. I wonder if this is a pattern that we should use elsewhere in atomate2? (If it's not already?)

Reply:
I think it's a nice pattern, although it was tricky to figure out. I hope that this addition will be a nice example for future workflows :)

Reply:
Thanks @Zhuoying, agreed :)

Reply:
This would be great! I think we could then easily connect the forcefields as well 😃